docs/section5/Rule5-43.md

```diff
@@ -0,0 +1,43 @@
+
+# Envelope - Rule 5-43  
+
+**Mandatory Rule:** True
+**Rule ID:** 5-43  
+**Rule Description:** Automatic fenestration shading devices shall not be modeled in the Baseline.  
+**Rule Assertion:** Baseline RMD = expected value  
+**Appendix G Section:** Section G3.1-5(f) Building Envelope Modeling Requirements for the Baseline building  
+
+**Manual Check:** None  
+**Evaluation Context:** Each Building  
+**Function Call:**  
+
+
+## Applicability Check:  
+- look at each zone: `for building in B_RMD.buildings:`
+    - look at each zone: `for zone in building...zones:`
+        - look at each surface: `for surface in zone.surfaces:`
+            - check if the surface is an exterior surface type: `if surface.adjacent_to == "EXTERIOR":`
+                - look at each subsurface: `for sub_surface in surface.subsurfaces: CONTINUE TO RULE LOGIC`
+    - if the code reaches here without going to the rule logic, the building has no exterior subsurfaces and the rule is not applicable: `NOT APPLICABLE`
+    
+    ## Rule Logic:  
+    - create a list automatic_shades_modeled and initialize it as an empty list: `automatic_shades_modeled = []`
+    - look at each zone: `for zone in B_RMD...zones:`
```

Comment:
nothing is being referenced from the zone or surface - we can go directly to subsurface

Reply:
I am going to surface, because we want exterior subsurfaces, but I removed the zone iteration